src/electron/electron-builder.json

```diff
@@ -19,6 +19,7 @@
     ],
     "files": [
       "third_party/badvpn/linux",
+      "third_party/outline-go-tun2socks/linux",
```

Comment:
It will be nice to have outline-go-tun2socks be in this repository too. That's client code and having it in a separate repository makes it harder to develop. We will need to figure out how to split it so that Intra doesn't come with it.

/cc @bemasc

Reply:
Outline-go-tun2socks has a pretty hacky build system so I'm scared to bring it into this repository, whose build system is already complicated enough.

Eventually, if we manage to switch to gVisor, we should be able to simplify the build system (no more `xgo`) and also establish a cleaner `Dialer`-like interface between the TCP engine and the Outline (or Intra) connection handlers.  Then maybe the Outline- and Intra- specific code could move into unified client repositories.  For now, though, I wouldn't mess with it.